core/templates/dev/head/pages/exploration_editor/EditorNavbarBreadcrumbDirective.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for showing Editor Navbar breadcrumb
+ * in editor navbar.
+ */
+
+oppia.directive('editorNavbarBreadcrumb', [function() {
+  return {
+    restrict: 'E',
+    templateUrl: 'inline/editor_navbar_breadcrumb_directive',
+    controller: [
+      '$scope', 'explorationTitleService', 'routerService',
+      'focusService', 'EXPLORATION_TITLE_INPUT_FOCUS_LABEL',
+      function(
+          $scope, explorationTitleService, routerService, focusService,
```

Comment:
One small nit: I suggest moving focusService to the next line, so that it matches the list of strings a couple of lines above. This makes it a bit easier to compare and make sure that the two lists are the same.